### PR TITLE
fix: event tags parsing

### PR DESCRIPTION
**Pull request by ryzizub**
fix for #8

### CODE REVIEW

File: test/event_test.dart [hunk 1]
```diff
@@ -124,7 +124,8 @@ void main() {
           ],
           [
             "p",
-            "052acd328f1c1d48e86fff3e34ada4bfc60578116f4f68f296602530529656a2"
+            "052acd328f1c1d48e86fff3e34ada4bfc60578116f4f68f296602530529656a2",
```
Comment (ethicnology) [inline test/event_test.dart]: we should fix up this change since it is not necessary

Comment (ethicnology) [reply test/event_test.dart]: I tried to fixup your commit, but not sur it worked…

Can you fixup and push this commit without changes on `test_event.dart`

Comment (ryzizub) [reply test/event_test.dart]: done 👍

**Comment by ethicnology** [comment]
@ryzizub Thank you, may i ask for a unit test that verify that your change is handling this particular case for a future refactoring (https://github.com/ethicnology/dart-nostr/issues/11)

**Comment by ryzizub** [comment]
@ethicnology added in https://github.com/ethicnology/dart-nostr/pull/9/commits/a8677207ad0264b17df2f739374faf65b8977f79

**Comment by ryzizub** [comment]
@ethicnology updated the tests, actually find out that the problem was when you decoded the json

**Review by ethicnology — DISMISSED** [review]

**Comment by ethicnology** [comment]
@ryzizub Thanks for your contribution :rocket:

**Pull request merged**